acrarium/src/main/kotlin/com/faendir/acra/service/MailService.kt

```diff
@@ -160,6 +164,9 @@ class MailService(
                     try {
                         val message = MimeMessageHelper(mailSender.createMimeMessage(), true)
                         message.setTo(it)
+                        if(StringUtils.isNotBlank(senderAddress)) {
+                            message.setFrom(senderAddress!!)
+                        }
```

Comment:
kotlin style: `senderAdress?.takeIf{ it.isNotBlank() }?.let { message.setFrom(it) }`